backend/src/main/java/dev/projects/backend/controller/PersonController.java

```diff
@@ -0,0 +1,121 @@
+package dev.projects.backend.controller;
+
+import dev.projects.backend.collection.Person;
+import dev.projects.backend.dto.PersonDTO;
+import dev.projects.backend.enums.LoginRole;
+import dev.projects.backend.service.PersonService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api")
+public class PersonController {
+
+    @Autowired
+    private PersonService personService;
+
+    //CREATE
+    @PostMapping("/person/master-admin")
+    public ResponseEntity<String> checkAndCreateMasterAdmin(){
+        List<Person> personList = getPersonsByRole(LoginRole.ADMIN);
+        if(personList.size()!=0) {
+            for (int i=0; i<personList.size(); i++){
+                if(personList.get(i).getUserName().equals("${ENV_UN_Admin}")){
+                    return ResponseEntity.ok("MasterAdmin already exists.");
+                }
+            }
+        }
+        PersonDTO masterAdmin = new PersonDTO();
+        masterAdmin.setUserName("${ENV_UN_Admin}");
+        masterAdmin.setPassWord("${ENV_PW_Admin}");
+        masterAdmin.setLoginRole(LoginRole.ADMIN);
+        saveAdmin(masterAdmin);
```

Comment:
why calling endpoint method? either implement it in "checkAndCreateMasterAdmin" or use a Service

Reply:
want to handle the master admin creation from the frontend if needed